src/domains/ecommerce-app/api/ecommerce-transactions-service/v2/_openapi.json.tpl

```diff
@@ -41,6 +41,15 @@
         ],
         "description": "Create a new transaction activating the payments notice by meaning of 'Nodo' ActivatePaymentNotice primitive",
         "parameters": [
+          {
```

Comment:
Why update this file?
These parameters are entered at the APIM level.

Reply:
this API are not currently used, but they will potentially be fields populated from an API key (user group or id) in the future

Reply:
You're right :D rollbacked witth https://github.com/pagopa/pagopa-infra/pull/1908/commits/f5b8e38d58b2ac796c604263dfc5e637460af332